src/libs/SidebarUtils.ts

```diff
@@ -37,11 +37,7 @@ Onyx.connect({
         // The report is only visible if it is the last action not deleted that
         // does not match a closed or created state.
         const reportActionsForDisplay = actionsArray.filter(
-            (reportAction, actionKey) =>
-                ReportActionsUtils.shouldReportActionBeVisible(reportAction, actionKey) &&
-                !ReportActionsUtils.isWhisperAction(reportAction) &&
-                reportAction.actionName !== CONST.REPORT.ACTIONS.TYPE.CREATED &&
-                reportAction.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
+            (reportAction) => ReportActionsUtils.shouldReportActionBeVisibleAsLastAction(reportAction) && reportAction.actionName !== CONST.REPORT.ACTIONS.TYPE.CREATED,
```

Comment:
The regression is caused by the removal of the CREATED condition. I missed the case where the last action is being used to display the last actor display name and the display name of the CREATED action is `__fake__`.
https://github.com/Expensify/App/blob/55c56bffbee8afd2fb5f5b2abfe30a92487bab52/src/libs/SidebarUtils.ts#L306-L310
<img width="213" alt="image" src="https://github.com/Expensify/App/assets/50919443/132109ce-0ed6-4aef-83e9-93f0178eeeb2">